addons/account/models/account_move.py

```diff
@@ -1782,6 +1782,20 @@ def _get_tax_cash_basis_base_account(self, line, tax):
     def _get_amount_tax_cash_basis(self, amount, line):
         return line.company_id.currency_id.round(amount)
 
+    def _get_tax_cash_basis_entry_date(self, move_date, newly_created_move):
```

Comment:
shouldn't this named `_set_tax_cash_basis_entry_date`? Since it doesn't get anything.